src/integrations/garmr.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  bulkhead,
+  circuitBreaker,
+  ConstantBackoff,
+  FailureReason,
+  handleAll,
+  IDefaultPolicyContext,
+  IPolicy,
+  noop,
+  Policy,
+  retry,
+  SamplingBreaker,
+  wrap,
+} from 'cockatiel';
+import { logger } from '../logger';
+import { isTest } from '../common';
+
+export class GarmrService {
```

Comment:
We need metrics here, not just logs

Reply:
Added metrics 7a5edc92c22e460fbc4503d7c8f4b73bf3fee5d7